Integrations/integration-DuoAdminApi.yml

```
@@ -0,0 +1,342 @@
+commonfields:
+  id: DUO Admin
+  version: -1
+name: DUO Admin
+display: DUO Admin
+category: Authentication
+image: data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHMAAAAmCAYAAAD3AKSiAAAGsUlEQVR42t1ca4hVVRQ+0kPTrIywlxaFJfkjKKGStK5z9t53bjUS1CgVM3Nec8eJikCMDHog/iuowASjSPwReWUmz9nnnPGOmf4Iqz/5osDAwVeiGQjlKx27feueM/SgZO69M3PXnQ2L4x3vrPOt9e21H2vvNYbli4Kj1W5by501SSB32YHYbgXiXS9WS2zddP9zce4ao8qWi2dNdLTwa8HmxmpvRyDfIH22b66gz9XqIhywL2otzLuqWpscf9FU8ounzSXkJyeQ26H7G8g6J1Su0yfme755o1Ftg6I9XjFbgqG1SaRKHp6d0EXihHLQDeWAG8n1+P8Wa1tmUiW46PuOlvs7a8DWtTUHHOoj0udqtQafq9ZFONBhD7YV1ZSK7PgkM8mO1BPQsc4N1QBsGkx9BH+lPtucLeXxmXzoRPI4nrETi86KiU17cAkvGVkJZRlovr+55PURULkTQK382rlXVEDmj14N2PBuPNWHpA+fV+Nz1brKOAI5MFwyM9syl4O8DrIb9hOWREd46feASBCdfB+//5MTy5VtvWp6pWSOqpBBSZTJbTTUjGcyXV/c50RqazkCYXe17yRe8luaKRD221otYUNmKiC0mUCetLWwxyOZmJsRjeok7BzBQFCl1A/v07DNhsy0xxHAP9DbVownMh0tXnVh12hNW4hSevY6/sNT2ZBJ4obJxG9psXw8kOn4chnsgV2j68suEIpn74txbiIbMtPJnubSwY5APNXIZFpaPdnZpy640dj4kSIUvL3Hisyh+cAN5bG2QN3RiGS6UdPtIPEo7BjDUU0m09Qm82lWZA453dbis0Yk0w7kp4iU+gSBlgN5nbmBFZk0uQPDIBzzaCOR6YXmAsLthPXxW9KJ1Fu8yISkjvYbiUxaiKQ66yKpb45SdLIi0w3LEXrG6Wue3QhktveadwPvaeBmMEVJjxWZJOk+alkjkOmE4mXgrbvPkGWiZ8yMzBRYIPqSBDV3MmUIvPUf0SLCJU5Q/pYVmeQwO5CHKMPRWmi9kiOZdATWtl5NcbQ4kGJjQajrNz1CZO5K93r1F4CCk847/sLZIPMylmSik7l+dhZ+ds6NePiNhnsrMJ8nI79FZJ7F8wwHAThgMecZJWMCRzLpaMvepB7EMMvGZwjGs+V9uteTmYHJ/E6nh4kACw1llHvkSGZet0wmfNx8hg4207CizE3tWtxmF9VMDkJYaPFDZCJK93Ekk/Bx85njy1soHfU1htlTAFx/CUnUaRpmUwIwn2e5DrOEl4XfMMyesgOxYchhfBZAoTzv9Yp7iAA3kP1Y/rNbAFla3mVr+TunBZCtxQustiYpjsP5grh2iAAAZbc1aS1krobfDnLamlBem2HSQBb/dnLf3tmf5Zk00DIGXi5E/kI3+dil84Dnlb8uRmWwn6P8J0MycbOAUTqvyC7R7kbyLA5c5xhpS/aaagsA8yMTiQ06GGCRaA/MLoZHYCIEpAn/vJIhnuVIZnowHUBnXYdYPI8tLarprA6nvTh70dXSNP7VaF+H7+z1Nmf5nWf65kLgruvhtBuKVfyujQSih4bV/7yp5ze1ciQTbYLty431iM70ztEh4JnOhkw4iebLE5S8Nv6nlZI8bQE9kRuZRjmtF6njY7pNCYnMcu3LM3yuWoaSru9fTEBduuV7sje7kdyHu6nsrlo6gVhcnibCsfFjV5IkWMPoEnSyrLZ883VjmM0L1VxM+j+DUHaXoK1AvpZcgh6T2xghrSVYkAlC0oIac6VRYbM2mQvg2KPoCKzITFfebyYVb6NXnuBqGVmfZ65jUjiUpcn7N9sX3UaVrd0378WQ+12ymmNWOIQ9H4bcX0cyO5TWcIJQsZYikkFJHwDBmXjXDisQDxk1tu5w/jQiCPrOl0kK+ZT00akKcH2V76eOm61pBKMOC12HbF911LWkj8CQA6lXIZJ+sEPRnZSkjVyzQgy7kdLQf4EMT06A6l9smx6uLwWu72E/YSF/DMtn9H2yhdYH4Obtdt106/Dq7AO120uql2qWtBBoqAye5Igby4ITicV00mCMYnN10wNeLN9xIrkHZJ4jh3RvfwyrPvVxuoX4oOuLXNW2QR+ReaDSMniyG3havUhugH8Ok1+GinDxmXQnFeb4Wbq9OemG4ks7Ui95cW5GZT1biyIMPeJo83BNEphHbC32I9I3lv/Ygm6e5xay1xtj3HKICDs25zioLEPHWk1/mCI5GxWrECXHqrYPPoKNO1r03MnVTw2PT6ODdwSQa2l0cviLdMNnW/Hv5XiHKF//qLL9CZ4cb+xabaaJAAAAAElFTkSuQmCC
+description: |-
+  DUO for admins.
+  Must have access to the admin api in order to use this
+configuration:
+- display: Host
```

Let's call it "API Hostname" as used in DUO's terminology

 ```suggestion
- display: API Hostname
```